src/macros/macros.js

```diff
@@ -39,9 +40,9 @@ module.exports = class Macros {
       // console.log(e)
     })
 
-    KeyboardJS.on("right", (e) => {
+    KeyboardJS.on("right", (_) => {
       document.dispatchEvent(
-        new Event("autocomplete/right")
+        new Events.Event("autocomplete/right")
```

Comment:
This is not correct, we are using a browser [`Event`](https://developer.mozilla.org/en-US/docs/Web/Events)
not an EventEmitter event